src/docs/truffle/testing/writing-tests-in-javascript.md

```diff
@@ -170,13 +170,14 @@ contract("2nd MetaCoin test", async accounts => {
     // Get initial balances of first and second account.
     const account_one = accounts[0];
     const account_two = accounts[1];
+    let balance;
```

Comment:
I would vote that this `let` just gets put on line 180 with the assignment rather than putting it separately up here.

Reply:
'balance' was used more than once, hence initializing it separately is quite clean.

Reply:
Hmm, I still don't really like it but won't hold it back for that!

Reply:
My pleasure, hopefully more collaborations to come.